src/jit/compiler.h

```diff
@@ -1329,11 +1374,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     DWORD expensiveDebugCheckLevel;
 #endif
 
-#if FEATURE_MULTIREG_RET
+#if FEATURE_MULTIREG_RET 
     GenTreePtr               impAssignStructClassToVar(GenTreePtr op, CORINFO_CLASS_HANDLE hClass);
 #endif // FEATURE_MULTIREG_RET
 
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
My suggestion is that in methods like IsHfa(hClass) - #ifdef could be moved inside the routine with #else portion of the logic always returns false.  This way we can avoid Jit-EE interface call overhead as well along with lesser #ifdef code.